features/management_classes/editClass.feature

```diff
@@ -15,4 +15,27 @@ Feature: I want to edit a class
 		Then <userRole> should see that class with name '<newCustomClassName>' and teacher lastname '<teacherLastname>' is visible
 		Examples:
 			| userRole | username                        | password        | customClassName | newCustomClassName | schoolYear | teacherLastname |
-			| teacher  | karl.teacher.qa@schul-cloud.org | Schulcloud1qa!  | 8a              | 4d                 | 2021/22    | Herzog          |
\ No newline at end of file
+			| teacher  | karl.teacher.qa@schul-cloud.org | Schulcloud1qa!  | 8a              | 4d                 | 2021/22    | Herzog          |
+
+	@editClassAddStudentTeacherAndDeleteStudentTeacher
+	Scenario Outline: As a user, I want to be able to edit a class
+		When <userRole> logs in with email '<username>' and password '<password>'
+		And <userRole> goes to management
+		And <userRole> goes to class management
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher1Lastname>' is visible
+		When <userRole> chooses class with name '<className>' and click managemnt class
+		And <userRole> adds a teacher to class with name '<teacher>'
+		And <userRole> adds a student to class with name '<student>'
+		And <userRole> clicks on Save-changes in class button
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher1Lastname>' is visible
+		Then <userRole> should see that class with name '<className>' and teacher lastname '<teacher2Lastname>' is visible
```

Comment:
duplicated step

Reply:
the same step but different parameter is used (teacher)